pkgs/by-name/wi/widevine-cdm/x86_64-linux.nix

```
@@ -1,29 +1,40 @@
 {
   lib,
   stdenv,
-  fetchzip,
+  fetchurl,
+  go-crx3,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
+  # Generated by update.py (don't edit this file manually)
   pname = "widevine-cdm";
   version = "4.10.2891.0";
 
-  src = fetchzip {
-    # The download 404s
-    url = "https://web.archive.org/web/20250725071306/https://dl.google.com/widevine-cdm/4.10.2891.0-linux-x64.zip";
-    hash = "sha256-ZO6FmqJUnB9VEJ7caJt58ym8eB3/fDATri3iOWCULRI=";
-    stripRoot = false;
+  src = fetchurl {
+    url = "https://edgedl.me.gvt1.com/edgedl/release2/chrome_component/aclxnidwwkj5di3vduduj2gqpgpq_4.10.2891.0/oimompecagnajdejgnnjijobebaeigek_4.10.2891.0_linux_b4hin3q5s66ws2322cyyfp35lu.crx3";
+    hash = "sha512-8S/4FRQQmQAO9wJ7QGJg8KUVxUQgm5/oEl6l9N2/WGAdMCIqxBoBPrCkPadrpwQUpK0WYX2fIZ8Zzfx3lNm8lA==";
   };
 
+  nativeBuildInputs = [ go-crx3 ];
+
+  unpackPhase = ''
+    unpackDir="src"
+    cp "$src" "$unpackDir".crx # go-crx3 doesn't like .crx3 extensions
+    crx3 unpack "$unpackDir".crx
+
+    cd "$unpackDir"
+  '';
+
   installPhase = ''
     runHook preInstall
 
     install -vD manifest.json $out/share/google/chrome/WidevineCdm/manifest.json
-    install -vD LICENSE.txt $out/share/google/chrome/WidevineCdm/LICENSE.txt
-    install -vD libwidevinecdm.so $out/share/google/chrome/WidevineCdm/_platform_specific/linux_x64/libwidevinecdm.so
+    install -vD LICENSE $out/share/google/chrome/WidevineCdm/LICENSE.txt
+    install -vD _platform_specific/linux_x64/libwidevinecdm.so $out/share/google/chrome/WidevineCdm/_platform_specific/linux_x64/libwidevinecdm.so
 
     runHook postInstall
   '';
 
   meta = import ./meta.nix lib;
+  passthru.updateScript = ./update-x86_64.py;
```

```suggestion
  passthru.updateScript = ./update-x86_64.py;

  meta = import ./meta.nix lib;
```
somewhere in the contributing it is written that meta should be last